app/importers/pure_publication_importer.rb

```diff
@@ -111,9 +111,11 @@ def record_type
   private
 
     def pub_attrs(publication)
+      subtitle = publication['subTitle'].try('[]', 'value')
+      subtitle = subtitle.present? ? ": #{subtitle}" : ''
```

Comment:
I know this is a little picky but can you refactor this into its own private method?

Otherwise, this all looks good!

Reply:
oh yeah that makes sense, not picky at all :)